src/panels/Scene/SceneGraphNode/OpenInNewWindowButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { ActionIcon } from '@mantine/core';
+
+import { usePropertyOwner } from '@/hooks/propertyOwner';
+import { OpenInNewIcon } from '@/icons/icons';
+import { Uri } from '@/types/types';
+import { displayName } from '@/util/propertyTreeHelpers';
+import { useWindowLayoutProvider } from '@/windowmanagement/WindowLayout/hooks';
+
+import { SceneGraphNodeView } from './SceneGraphNodeView';
+
+interface Props {
+  uri: Uri;
+}
+
+export function OpenInNewWindowButton({ uri }: Props) {
```

Comment:
```suggestion
export function OpenSgnInNewWindowButton({ uri }: Props) {
```
To make it clear it is a scene graph node:)

Reply:
Make we want to make this button more general. We already have the same button in the skybrowser panel, for popping out the wwt window. If we pass the data passed to addWindow as a prop, I think that could potentially be a useful component to have 🤔

Reply:
Good point!  Maybe we could do a refactor after the PRs are in? 

Would be nice to have all the "open in new window" buttons look and work the same :)

Reply:
That one is already in master, but sure we can do a refactor later